WalletWasabi/Services/WalletService.cs

```diff
@@ -1137,101 +1137,79 @@ public Operation(Script script, Money amount, string label)
 			// 4. Get and calculate fee
 			Logger.LogInfo<WalletService>("Calculating dynamic transaction fee...");
 
-			FeeRate feePerBytes = (feeTarget.HasValue) ? Synchronizer.GetFeeRate(feeTarget.Value) : feeRate;
+			FeeRate feePerBytes = (feeTarget is int v) ? Synchronizer.GetFeeRate(v) : feeRate;
+			feePerBytes = FeeRate.Max(feePerBytes, new FeeRate(1m));
```

Comment:
Could you add a comment to this fee rate correction, please? Because it looks evident but the previous code had a comment so, it seems the original author thought it was important and I think it doesn't hurt.